### ISSUE TITLE
Images need to be resized.

### ISSUE THREAD

**Issue by dummey** [mpmenne/cs50-connection#2]
Currently, the full size images are being loaded. It should instead be resized thumbnails so that they are smaller and loads faster.

**Comment by rennaw** [comment]
Hey. I ended up at E.R. Good news I said chest pain and was moved to the
front of the line.

Not so good news...I have blood in my sputum and a dark spot on my lungs. I
have pneumonia still.

I have to go back at 7 in the a.m. for a different kind of chest scan, and
hopefully results of the spit under microscope which will either confirm or
refute bacterial pneumonia.  They really doubt bacterial pneumonia as I
only have (had at the time) a temp of 99.3 and if it is bacterial the temp
is normally high. But my white count is crazy high so they are not sure.

They wanted to admit me but I kinda threw a fit. I do not feel sick I just
hurt a little.  They gave me some pain meds and let me go tonight pending
test results in the a.m.

So bottom line as I understand it is I either have viral or bacterial
pneumonia which I should know for sure sometime tomorrow.  Either way I
still intend on attending Friday.   I didn't work this hard to let a little
'ugly' stop me.

On that note however I am going to bed. Fill you guys in tomorrow when I
know more. Thanks for checking in!

Dana

**Comment by rennaw** [comment]
Sorry!  This went out to wrong people!

**Comment by mpmenne** [comment]
Oh man hope you get better!

**Comment by mpmenne** [comment]
Ricky good call with the image resize... current website is crashing my weeeeak-ass mac  :-)

**Comment by mpmenne** [comment]
okay just resized the images... I think some of are still 100K, but it loads a lot better. 

They could probably be a lot smaller but I wanted them to still be usable.  

Let me know your thoughts and I'll close it out.  Thanks again for the bug request!
